CMakeLists.txt

```diff
@@ -138,7 +135,9 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/src/anbox/build/config.h.in
 
 add_subdirectory(external)
 add_subdirectory(src)
-add_subdirectory(tests)
+if (CMAKE_TESTING_ENABLED)
```

Comment:
This disables the tests by default which breaks our CI. By default tests should be enable and if somebody wants them disabled (e.g for cross compilation) the flag can be set.

Reply:
Most opensource projects keep tests disabled by default.

Reply:
Why would tests be disabled by default? Whenever you build something you always want it to be tested. Otherwise you cannot trust what you've build.

Reply:
I replaced CMAKE_TESTING_ENABLED to BUILD_TESTING defined in cpu_features.